.gitignore

```diff
@@ -22,3 +22,9 @@ kpt/kpt
 
 # Emacs temp files
 *~
+
+# VS Code individual configurations
+.vscode/
+
+# local tmp directory (in case you download packages here)
+tmp/
```

Comment:
Not sure if it matters, but there is no newline at the end of the file.

Reply:
good feedback.  Fixed.